config/schemas/config.yaml

```diff
@@ -7409,6 +7409,53 @@ properties:
         examples:
           - default-timeout-minutes: "30"
         type: object
+  gpu:
+    title: GPU Config
+    description: Configure the GPU Operator and its dependencies
+    type: object
+    properties:
+      enabled:
+        title: GPU Operator Enabled
+        default: false
+        type: boolean
+      operator:
+        title: GPU Operator Config
+        description: Configure GPU Operator
+        type: object
+        properties:
+          resources:
+            $ref: '#/$defs/kubernetesResourceRequirements'
+          tolerations:
+            $ref: '#/$defs/kubernetesTolerations'
+          affinity:
+            $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+      nodeFeatureDiscovery:
+        title: Node Feature Discovery Config
+        description: Configure Node Feature Discovery
+        type: object
+        properties:
+          master:
+            title: Node Feature Discovery master Config
+            description: Configure Node Feature Discovery masters
+            type: object
+            properties:
+              resources:
+                $ref: '#/$defs/kubernetesResourceRequirements'
+              tolerations:
+                $ref: '#/$defs/kubernetesTolerations'
+              affinity:
+                $ref: '#/$defs/io.k8s.api.core.v1.Affinity'
+          worker:
+            title: Node Feature Discovery worker Config
+            description: Configure Node Feature Discovery workers
+            type: object
+            properties:
+              resources:
+                $ref: '#/$defs/kubernetesResourceRequirements'
+              tolerations:
+                $ref: '#/$defs/kubernetesTolerations'
```

Comment:
Question: Do we need to specify the defaults from the config here? i.e. the `nvidia.com/gpu:NoSchedule` for this one

Reply:
Hmm, that's a good point. But I couldn't find any tolerations in the config that we define defaults for, even if they have defaults in the config (see e.g. `falcoExporter`: defaults [in the config](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.43.0/config/common-config.yaml#L201), but [not in the schema](https://github.com/elastisys/compliantkubernetes-apps/blob/v0.43.0/config/schemas/config.yaml#L3263)), so I'm not sure what the standard is here?